src/screens/Settings/SettingsScreen.js

```diff
@@ -16,25 +16,28 @@ import images from '../../constants/images';
 
 import styles from './SettingsScreen.style';
 
-import SettingsItem from '../../components/SettingsItem.js';
+import SettingsItem from '../../components/SettingsItem';
 import { theme } from '../../theme.js';
+import { openURL } from '../../helpers/index.js';
+import { HELP_URL } from '../../constants/url.js';
 const settingsData = [
-  {
-    text: i18n.t('SETTINGS.AVAILABILITY'),
-    checked: false,
-    iconSize: 'small',
-    itemType: 'toggle',
-  },
-  {
-    text: i18n.t('SETTINGS.PUSH'),
-    checked: false,
-    iconSize: 'small',
-    itemType: 'toggle',
-  },
+  // {
```

Comment:
Remove this lines.

Reply:
Let us not commented code, if it is not being used, let us remove and re-introduce whenever required.